_sass/_layout.scss

```diff
@@ -331,6 +325,36 @@
         margin-top: 40px;
     }
 
+    .activity-page {
+      padding: 0 0 20px 0;
+      background-color: #DAE3E7;
+
+      .alert {
+        border-radius: 0;
+      }
+
+      .post-content {
+        .col-md-9 {
+          width: 74%;
+          margin-left: 1%;
+          background-color: #FFFFFF;
+          padding: 20px;
+        }
+
+        @media(max-width: 768px) {
+          .col-md-9 {
+            width: 100%;
+          }
+        }
+        .col-md-3 {
+          width: 24%;
+          margin-right: -1%;
+          background-color: #FFFFFF;
```

Comment:
given the fact that we want to adopt to mozilla style colors in the future, could you please add this as a variable?